nservicebus/sqlserver/configuration.md

```diff
@@ -37,6 +37,11 @@ Combined with a named connection in the `connectionStrings` node of the `app.con
 
 snippet:sqlserver-named-connection-string-xml
 
+### Via a custom connection factory
+
+It is also possible to pass to transport a custom factory method, which will provide connection strings at runtime:
```

Comment:
This doesn't flow well with the document above. All of the preceding sections have the form

> Via something
> by some process

Should this just be:

> By passing the transport a custom factory method which will provide connection strings at runtime: